apps/emqx_bridge_kinesis/src/emqx_bridge_kinesis_impl_producer.erl

```diff
@@ -0,0 +1,248 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022-2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_bridge_kinesis_impl_producer).
+
+-include_lib("emqx/include/logger.hrl").
+-include_lib("emqx_resource/include/emqx_resource.hrl").
+-include_lib("snabbkaffe/include/snabbkaffe.hrl").
+
+-define(HEALTH_CHECK_TIMEOUT, 15000).
+-define(TOPIC_MESSAGE,
+    "Kinesis stream is invalid. Please check the logs, check if the stream "
+    "exist in Kinesis and if the service account has permissions to use them."
```

Comment:
Looks like this message needs to be adjusted for kinesis

Reply:
Ack